src/styles/sidebar/components/thread.scss

```diff
@@ -0,0 +1,50 @@
+@use "../../variables" as var;
+
+.thread {
+  &--reply {
+    margin-top: 0.5em;
+    padding-top: 0.5em;
+  }
+  display: flex;
+
+  &__collapse {
+    border-left: 1px dashed var.$grey-3;
+    margin: 0.25em;
+    margin-top: 0;
+    cursor: auto;
```

Comment:
I'll call this out. I did this to override the `pointer` cursor that applies across the entire annotation card. I don't want to imply that this area is clickable—only the button is.